src/core/time.py

```diff
@@ -0,0 +1,9 @@
+class DeltaTime:
+    value = 0
+    @staticmethod
+    def process(pygame_delta):
+        DeltaTime.value = pygame_delta / 1000
+
+    @staticmethod
+    def get():
+        return DeltaTime.value
```

Comment:
Acho que aqui fica mais idiomático usar class methods ao invés de static methods (eles pegam a classe como argumento):
```suggestion
    @classmethod
    def process(cls, pygame_delta):
        cls.value = pygame_delta / 1000

    @classmethod
    def get(cls):
        return cls.value
```

Reply:
Vou aplicar, eu fiz estático porque não conheço a estrutura do código de vocês ainda